dss-demo-webapp/pom.xml

```diff
@@ -266,6 +270,19 @@
 			<artifactId>logback-classic</artifactId>
 		</dependency>
 
+		 <dependency>
+            <groupId>ch.qos.logback.contrib</groupId>
+            <artifactId>logback-jackson</artifactId>

```

Comment:
Is this correct?
```suggestion
            <artifactId>logback-json</artifactId>
```

Reply:
<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
		 <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
            <layout class="ch.qos.logback.contrib.json.classic.JsonLayout">
                <timestampFormat>yyyy-MM-dd'T'HH:mm:ss.SSSX</timestampFormat>
				<jsonFormatter class="**ch.qos.logback.contrib.jackson.JacksonJsonFormatter"**/>
            </layout>
		</encoder>
	</appender>
Yes - this is used by the logback config